kube-runtime/src/watcher.rs

```diff
@@ -919,3 +919,13 @@ impl Backoff for DefaultBackoff {
         self.0.reset()
     }
 }
+
+#[cfg(test)]
+mod tests {
+    #[test]
+    fn backoff_start_interval() {
+        let default_backoff = super::DefaultBackoff::default().0;
+        let default_backoff = default_backoff.get_backoff();
+        assert_eq!(default_backoff.current_interval, default_backoff.initial_interval);
+    }
+}
```

Comment:
tbh, i wouldn't worry about this. i don't think this test super helpful, and it's another public interface for it.

Reply:
Yeah that's fair enough, reverted.